templates/default/etc/systemd/system/knotx.service.erb

```diff
@@ -20,7 +20,7 @@ ExecStart=<%= @java_home %>/bin/java \
   -Dvertx.cli.usage.prefix=knotx \
   -classpath '<%= @conf_dir %>:<%= ::File.join(@lib_dir, '*') %>' \
   $KNOTX_EXTRA_OPTS \
-  io.vertx.core.Launcher run-knotx
+  io.vertx.core.Launcher run-knotx $KNOTX_INSTANCE_OPTS
```

Comment:
Just sanity checking: If we put the variable in the same line, won't systemd throw warns and errors there is some empty var? Shouldn't it be put in new line?

Reply:
There is no warns when running `kitchen converge`. So it should make no problem.